src/d3a/models/strategy/__init__.py

```diff
@@ -231,6 +232,14 @@ def __init__(self):
 
     parameters = None
 
+    @property
+    def simulation_config(self) -> SimulationConfig:
```

Comment:
> If anything changes in the interface of any of the intermediate objects, everything breaks.

But the wrapping property will not help with that, it will fail too, right?

Reply:
@hannesdiedrich The property wouldn't "save" us from this, but it would make our life easier. It would allow us to fix the breaking bug by just modifying one line (the content of the property). If we didn't have it, we would need to change all the calls in the entire codebase.
Also, suppose we use properties to alias long chains of calls. Once we start doing it, we'll find out that even the content of our properties won't be a long chain of calls. That's because it will contain calls to aliases. E.g.:
```python
@property
def simulation_config(self):
    return self.area.config

@property
def grid_fee_type(self):
    # Here we abbreviated even more. It would have been `self.area.config.grid_fee_type`
    return self.simulation_config.grid_fee_type
```

Reply:
ok, convinced!